ofronds_internals/exercise.mli

```diff
@@ -0,0 +1,18 @@
+type t [@@deriving sexp]
+
+val of_file :
+  string -> name:string -> (t, [ `File_not_found | `Name_absent ]) result
```

Comment:
Good catch; that's an unnecessary hold-over from a previous iteration that didn't have `Set.of_file`.

I was mostly copying `rustlings` here and thinking about providing equivalent support for `rustlings run <exercise>`. Even in that case, it might be better / neater to construct the entire `Set.t` and then `find` in it; I'm not sure.

Reply:
All right, I should have a look at the `rustlings` implementation at some point. I was just curious here if you had something concrete in mind.